### PR TITLE
Disclaimer for JENKINS-44747 change.

### PR DESCRIPTION
**Pull request by abayer**
This is on top of/subsumes https://github.com/jenkinsci/throttle-concurrent-builds-plugin/pull/50, fixing [JENKINS-44747](https://issues.jenkins-ci.org/browse/JENKINS-44747)

cc @reviewbybees

### CODE REVIEW

**Comment by abayer** [comment]
Whoops, didn't mean to push to jenkinsci. My remotes were set up wrong. Sigh. Sorry!

File: README.md [hunk 1]
```diff
@@ -77,6 +77,13 @@ How does it work?
 * Throttling of Pipeline steps in `throttle()` will take other throttling logic like job properties in Pipeline and other job types.
 * If the specified category is missing, `throttle()` execution will fail the run.
 
+#### Warning regarding restarting master
+
+Due to a deadlock (as described in [JENKINS-44747](https://issues.jenkins-ci.org/browse/JENKINS-44747)), a change has
```
Comment (oleg-nenashev) [inline README.md]: Add `:exclamation:` to the beginning

File: README.md [hunk 2]
```diff
@@ -77,6 +77,13 @@ How does it work?
 * Throttling of Pipeline steps in `throttle()` will take other throttling logic like job properties in Pipeline and other job types.
 * If the specified category is missing, `throttle()` execution will fail the run.
 
+#### Warning regarding restarting master
+
+Due to a deadlock (as described in [JENKINS-44747](https://issues.jenkins-ci.org/browse/JENKINS-44747)), a change has 
+been made which can theoretically result in throttle categories being ignored in running Pipelines immediately after the 
+Jenkins master has been restarted. This will be investigated further, but was considered a necessary change in order to 
+resolve the deadlock scenario.
```
Comment (oleg-nenashev) [inline README.md]: Issue link?

File: README.md (hunk 1 above)
Comment (abayer) [inline README.md]: Will do.

File: README.md (hunk 2 above)
Comment (abayer) [inline README.md]: There isn't one? =)

Comment (abayer) [inline README.md]: There is now - https://issues.jenkins-ci.org/browse/JENKINS-44756 - adding.

**Pull request merged**